content/en/docs/setup/install/multicluster/gateways/index.md

```diff
@@ -63,16 +63,6 @@ Cross-cluster communication occurs over the Istio gateways of the respective clu
 1. Run the following commands in **every cluster** to deploy an identical Istio control plane
     configuration in all of them.
 
-    {{< tip >}}
-    Make sure that the current user has cluster administrator (`cluster-admin`) permissions and grant them if not.
-    On the GKE platform, for example, the following command can be used:
```

Comment:
I never liked these platform specific instructions. This part can definitely go.